src/SFA.DAS.LevyTransferMatching.Web/Orchestrators/PledgeOrchestrator.cs

```diff
@@ -220,6 +225,82 @@ public async Task<LocationSelectViewModel> GetLocationSelectViewModel(LocationSe
             };
         }
 
+        public async Task<byte[]> GetPledgeApplicationsDownloadModel(ApplicationsRequest request)
+        {
+            var result = await _pledgeService.GetApplications(request.AccountId, request.PledgeId);
+
+            var pledgeAppModel = new PledgeApplicationsDownloadModel
+            {
+                Applications = result.Applications?.Select(app =>
+                {
+                    var affordability = GetAffordabilityViewModel(app.Amount, app.PledgeRemainingAmount,
+                        app.NumberOfApprentices, app.MaxFunding, app.StandardDuration, app.StartDate);
+
+                    return new PledgeApplicationDownloadModel
+                    {
+                        DateApplied = app.CreatedOn,
+                        Status = app.Status,
+                        ApplicationId = app.Id,
+                        PledgeId = app.PledgeId,
+                        EmployerAccountName = app.EmployerAccountName,
+                        HasTrainingProvider = app.HasTrainingProvider,
+                        Sectors = app.Sectors ?? new List<string>(),
```

Comment:
Could be bad data, but this field was blank for me when I ran it locally.

Reply:
yeah it was missing from the response model so has been added